data_model/src/expression.rs

```diff
@@ -325,11 +348,13 @@ pub struct ContextValue {
 
 impl ContextValue {
     /// Number of underneath expressions.
+    #[inline]
```

Comment:
Not usually necessary for `const` functions, **but** because this function is likely to also be called in run-time, this might not be a bad idea.